`make build` failing

### Description

`make build` fails the the error:
```
[+] Building 0.0s (0/0)                                                                                            docker:desktop-linux
invalid duplicate ID default
make: *** [build] Error 1
```

### Steps to Reproduce

1. Run `make build`
2. See error.

Issue verified by @jak574 and @swyatt7 

### Expected Behavior

Builds Docker containers.

--- a/Makefile
+++ b/Makefile
@@ -157,7 +157,7 @@ stop_all: check_prod ## Stop all containers
 	@$(DOCKER_COMPOSE) down -v
 
 build: check_prod ## Build the containers (does not run them)
-	@DOCKER_BUILDKIT=1 $(DOCKER_COMPOSE) build --ssh default --build-arg APP_ENV=$(ENV)
+	@DOCKER_BUILDKIT=1 $(DOCKER_COMPOSE) build --build-arg APP_ENV=$(ENV)
 
 restart: ## Restarts the app container
 	@$(DOCKER_COMPOSE) restart

